pinot-common/pom.xml

```diff
@@ -190,6 +190,10 @@
       <groupId>org.restlet.jse</groupId>
       <artifactId>org.restlet</artifactId>
     </dependency>
+    <dependency>
```

Comment:
I also see imports of 'org.codehaus.jackson' in various other parts of the code. We should pick one and use that uniformly.